### PR TITLE
Jabba performance adjust

### PR DESCRIPTION
**Pull request by jahtim**
5% fps increase if you have 50 jabba barrels in your render distance. This change actually makes the item and text on the barrel not rendering when the player is far (4m default). and this could be configurable.

### CODE REVIEW

**Comment by draknyte1** [comment]
4M is far too low for a new default. 
Suggest 4m is the comment, leave default as default.

**Review by draknyte1 — CHANGES_REQUESTED** [review]
As above.

**Comment by jahtim** [comment]
Changed to distance of 100 as default

**Comment by jahtim** [comment]
Changed to distance of 100 as default

**Review by draknyte1 — APPROVED** [review]

File: src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public void preInit(FMLPreInitializationEvent event) {
 			
 			allowOreDictUnification = config.getBoolean("allowOreDictUnification", Configuration.CATEGORY_GENERAL, true, "If true, Jabba will try unificate 'ingot' 'ore' 'dust' and 'nugget' using oredict");
 
-			renderDistance = config.getFloat("renderDistance", Configuration.CATEGORY_GENERAL, 16f, 0f, 10000f, "Render Distance (square) for stack and text on barrel.");
+			renderDistance = config.getFloat("renderDistance", Configuration.CATEGORY_GENERAL, 10000f, 0f, 10000f, "Render Distance (square) for stack and text on barrel.");
```
Comment (hazzal) [inline src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java:132]: ~~Can we change the text for the config to describe that it is actually cubed (not squared) distance.~~
Please disregard, thankfully Glease corrected my wrong interpretation.

Comment (Glease) [reply src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java:132]: No. Check the code again. It is indeed squared.

Comment (hazzal) [reply src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java:132]: Apologies in that case. Please disregard my initial comment and thanks for correcting me Glease

**Comment by Dream-Master** [comment]
@TataTawa @draknyte1 @hazzal can we merge it ?

**Comment by draknyte1** [comment]
Yes.

**Pull request merged**